.github/workflows/maven.yaml

```
@@ -0,0 +1,60 @@
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Java CI
+
+on:
+  push:
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 19: did not find expected ',' or ']'

(syntax-check)